### PR TITLE
Sign Up Page UI -- ERROR = OmniAuth direction

### PR DESCRIPTION
**Pull request by ahmedg10**
## Pull Request Title
Include a short, descriptive title regarding the issue the request intends to resolve.

## Reviewer Guidance
Tested and updated with main. Works as expected. Need to add new system tests for user registration since we have now removed username/email/password authentication.

## Demo
To test, run yarn run start, navigate to /users/sign_up and sign in with Github. To edit (or if you're already signed in) navigate to users/edit and hit cancel my account. This should redirect you to index home.

### CODE REVIEW

**Comment by andrewjumanca** [comment]
@ahmedg10 Will take a look at this and test it on my end to see if I can rule out the system-tests. Also just for next time's branch naming, it's {last name}-{repository name, so usually will be platform}-{issue # (64 in this case)}. UI looks clean B), nice work overall!

**Comment by GitRepsOfficial** [comment]
@ahmedg10 There's a couple things you'll need to get this working:

1. Wait until #65 is merged, then rebase to include those changes. It contains a [fix](https://github.com/GitRepsCommunity/platform/blob/41e68b4c3035aaa659d08d5cbdd4e4c1f755c75e/app/models/user.rb#L15) in the `from_omniauth` that is keeping user records from getting created.
2. Update the [user_registrations_test](https://github.com/GitRepsCommunity/platform/blob/c0a0fc4df82863adc8477b479abccc1df0764d37/test/system/user_registrations_test.rb#L5C7-L5C28) and remove all of the existing tests for the old registration process. Then add something like this:

```ruby
  test 'Sign up with GitHub' do
    visit new_user_registration_url

    click_on 'Sign up with GitHub'
    assert_redirected_to %r(\Ahttps://github.com/login)
  end
```

**Comment by andrewjumanca** [comment]
Tested and updated with main. Works as expected. To test, run yarn run start, navigate to /users/sign_up and sign in with Github. To edit (or if you're already signed in) navigate to users/edit and hit cancel my account. This should redirect you to index home.

File: test/system/user_registrations_test.rb
Comment (GitRepsOfficial) [inline test/system/user_registrations_test.rb:6]: You'll need to wrap this in a test block for it to work:

```ruby
test 'Sign up with GitHub button' do
    visit new_user_registration_url
    ...
end
```

File: test/system/user_registrations_test.rb
Comment (GitRepsOfficial) [inline test/system/user_registrations_test.rb:7]: Strange that the url helper function `new_user_registration_url` doesn't work here anymore 🤔 This is good for now, but we might want to figure out how to fix that at some point since it seems like some assertions aren't working either.

**Review by GitRepsOfficial — APPROVED** [review]

File: .yarn/install-state.gz
Comment (GitRepsOfficial) [inline .yarn/install-state.gz:1]: Do we need this file?

**Review by GitRepsOfficial — CHANGES_REQUESTED** [review]

File: test/system/user_registrations_test.rb
Comment (andrewjumanca) [reply test/system/user_registrations_test.rb:7]: When I debugged using the Rails console, it would seem to correctly print out the new_user_registration_url and new_user_registration_path, however instead of the localhost URL it had some strange code as the prefix, and both values were followed by a nil value.

File: .yarn/install-state.gz
Comment (andrewjumanca) [reply .yarn/install-state.gz:1]: Must have been from all the rebuilding I had to do, will remove!

**Review by GitRepsOfficial — APPROVED** [review]

**Pull request merged**
